src/viam/media/video.py

```diff
@@ -34,6 +36,27 @@ def close(self):
         """Close the image and release resources. For RawImage, this is a noop."""
         return
 
+    def bytes_to_depth_array(self) -> List[array]:
+        """Decode the data of an image that has the custom depth MIME type ``image/vnd.viam.dep`` into
+        a standard representation.
+
+        Raises:
+            NotSupportedError: Raised if given an image that is not of MIME type `image/vnd.viam.dep`.
+
+        Returns:
+            List[array]: The standard representation of the image.
+        """
+        if self.mime_type != "image/vnd.viam.dep":
+            raise NotSupportedError("Type must be `image/vnd.viam.dep` to use bytes_to_depth_array()")
+
+        width = int.from_bytes(self.data[8:16], "big")
+        height = int.from_bytes(self.data[16:24], "big")
+        depth_arr = array("H", self.data[24:])
```

Comment:
I am surprised you dont have to specify the endianness of these unsigned shorts but you do for the other values.